src/datamanager/__main__.py

```
@@ -41,6 +42,22 @@ def _rel(iso: str) -> str:
     return f"{hours} h ago"
 
 
+def _validate_temoa_hash(temoa_hash: str) -> bool:
+    """
+    Validates that a temoa repo hash looks like a valid git commit hash.
+    accepts 4–40 hexadecimal characters (case-insensitive).
```

Ruff (0.14.0)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)